terminal/config.yaml

```diff
@@ -0,0 +1,14 @@
+
+# Before deploying the terminal, create a secret name $TERMINAL_CREDENTIAL 
+# with your "username:password".
+
+ingress:
```

Comment:
since this shouldn't be changed in okteto cloud, it should just be in values.yaml